src/app/components/Blog.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import Post from './Post';
+import WelcomeModal from './WelcomeModal';
+
+export default class Blog extends Component {
+  
+  render() {
+
+    const items = this.props.content.map((item, index) => {
+      return <Post 
+        key={index} 
+        title={item.title} 
+        text={item.text}
+        author={item.author}
+        date={item.date}
+        />;
+    })
+
+
+    return (
+
+      <div>
+        {this.componentDidMount()}
```

Comment:
```suggestion
        <WelcomeModal />
```
"вызвать метод в componentDidMount()", а вы вызвали сам componentDidMount() :))
Не знал, что так будет работать)))